specification/network/resource-manager/Microsoft.Network/2017-06-01/virtualNetworkGateway.json

```diff
@@ -237,7 +237,7 @@
           "200": {
             "description": "Request successful. The operation returns a list of VirtualNetworkGatewayConnection resource.",
             "schema": {
-              "$ref": "#/definitions/VirtualNetworkGatewayConnectionListResult"
```

Comment:
Looks like you are renaming ```VirtualNetworkGatewayConnectionListResult``` to ```VirtualNetworkGatewayListConnectionsResult``` 
might want to delete the other model definition too since its no longer used.